### DISCUSSION TITLE
KeeShare with reference and custom fields

### DISCUSSION THREAD

**Discussion by theincogtion** [keepassxreboot/keepassxc#10274] [Q&A]
I am starting with the KeeShare Folder and like to have an entry which references to my main database.
Is this even possible? If the user with which I share the entry cannot see the original entry in his database and references may fail?
Second question is about custom fields:
According to the original KeePass documentation it will not work:
"Referencing fields of other entries only works with standard fields, not with custom user strings. "
Is this the same with KeePassXC?
Looks like
O – Custom Attribute (SEARCH_IN only)
https://keepassxc.org/docs/KeePassXC_UserGuide#_entry_cross_reference

**Comment by droidmonkey** [comment]
References only work if the referenced entry is in the database. So references used in keeshare databases will not resolve unless the keeshare or parent database has the respective entry.
